server/agent/migrations/0013_agent_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.2.4 on 2025-08-12 09:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("agent", "0012_agent_corrupted"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="agent",
+            name="description",
+            field=models.TextField(default="Lorem ipsum."),
```

Comment:
I'm not sure if we want to add a default here :)

Reply:
That's the default only for migration, to fill existing rows

Reply:
Ah ok :)